src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php

```diff
@@ -311,9 +313,17 @@ private function loginHelper()
         }
 
         $token = $this->Request()->getParam('token');
-        $this->session->offsetSet('sgWebView', true);
+        Shopware()->Session()->offsetSet('sgWebView', true);
```

Comment:
Is using `$this->session` an issue in this case?

Reply:
@Rekhyt my bad, seems like I am not getting emails for this. 

The change is not vital to the fix, but as I was debugging this class, I noticed that the "injected" session and global session have different values set to them. Which is kinda bad. To avoid confusion in the future, I thought that it would be safer to assign to the global state (at least for sessions).